### ISSUE TITLE
admin metadata for batch ingest

### ISSUE THREAD

**Issue by ebenenglish** [marriott-library/newspaper_works#153]
**Descriptive Summary**
We want to allow admin users to set the following properties on objects create via batch ingest:
* depositor
* admin set
* visibility

We may want to set a default if no argument is provided?
* depositor: `RoleMapper.whois('admin').first`
* admin set: `AdminSet.where(id: "admin_set/default").first`
* visibility: `open`

All objects should have `resource_type` value of `["Newspapers"]`

`NewspaperTitle` and `NewspaperIssue` objects created should have the following properties created by default: 
* `date_uploaded` (should be same as `date_modified` time)
* `state` (should be `#<ActiveTriples::Resource:0x384fc30 ID:<http://fedora.info/definitions/1/0/access/ObjState#active>>`)

(Page objects created via NDNP batch already have `date_uploaded` and `state` values.)

**Rationale**
These properties are useful for asset management, and are needed to match the admin metadata properties of other objects in the repository created via single-item upload.

**Expected behavior**
As an admin user
When I am running a rake task to start a batch ingest
I should be able to pass an optional argument for depositor
And I should be able to pass an optional argument for admin set
And I should be able to pass an optional argument for visibility
And when I run the rake task
The ingested objects should have the supplied depositor (or the default)
And should belong to the supplied admin set (or the default)
And should have the supplied visibility (or the default)
And should have `["Newspapers"]` as the value of `resource_type`
And should have a `date_uploaded` property corresponding to the upload date
And should have a `state` property corresponding to `active`

**This ticket will be complete when:**
Functionality and user story described above is implemented and tested.

**Comment by seanupton** [comment]
The `ndnp-metadata` branch as of f8588c957f032e4fd2ebf4b35707c41b6dd367c7 implements functionality described in this issue.  (That branch is also being used for implementation of #154).

**Comment by ebenenglish** [comment]
Looks good so far.

One question -- in the specs, do we want to use `AdminSet::DEFAULT_ID` to find the AdminSet for comparison?

As in:
```
expect(asset.admin_set).to eq AdminSet.find(AdminSet::DEFAULT_ID)
```
instead of
```
expect(asset.admin_set).to eq AdminSet.find('admin_set/default')
```

Not a big deal either way, but might make specs slightly less brittle if future changes to admin set default id are made upstream in Hyrax?

**Comment by seanupton** [comment]
@ebenenglish yeah, that was an oversight on my part, intended to do just that.  Fixed in e3238389e7b9c7d2878eb89622699d0f6584fe51

_Only exception to this: test of CLI arguments passed has hard-coded name passed in fake ARGV, so matching that by string and not by someone else's constant makes sense._

**Comment by ebenenglish** [comment]
This behavior has been tested as of https://github.com/marriott-library/newspaper_works/commit/fffdbf676277e80c10b8db6232838c89df659f3d as is working as expected.

**Comment by ebenenglish** [comment]
Fixed via #164.
